### ISSUE TITLE
[Enahncement] Add auto refresh option + email?

### ISSUE THREAD

**Issue by jimger** [vloss3/wizz-aycf-route-finder#10]
Adding auto refresh option (for either a specific route or every route) along with saving results can be very useful for statistics in the future.

Potentially sending email (or maybe telegram or similar) as notifier could be helpful in tracking flights

**Comment by vloss3** [comment]
good idea, it can be done via Discord/Telegram webhooks so it doesn't require servers
